lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -50,11 +53,7 @@ def to_representation(self, course_overview):
                 }
             },
             'course_image': course_overview.course_image_url,
-            'course_about': reverse(
-                'about_course',
-                kwargs={'course_id': course_id},
-                request=request,
-            ),
+            'course_about': None if not course_about_url else "http:{}".format(course_about_url),
```

Comment:
Why are we stripping out the URL scheme (http/https), and then coercing it to `http` here?  Shouldn't we maintain the version specified, or failing that, require `https`?